src/main/java/se/sundsvall/mex/integration/casedata/CaseDataClient.java

```diff
@@ -25,69 +27,87 @@
 import generated.se.sundsvall.casedata.PatchErrandDTO;
 import generated.se.sundsvall.casedata.StakeholderDTO;
 import generated.se.sundsvall.casedata.StatusDTO;
-import se.sundsvall.mex.integration.casedata.configuration.CaseDataConfiguration;
 
 @FeignClient(name = CLIENT_ID, url = "${integration.casedata.url}", configuration = CaseDataConfiguration.class)
 public interface CaseDataClient {
 
 	/**
 	 * Updates a decision.
-	 * 
+	 *
```

Comment:
Since there is a javadoc here, we should keep it up-to-date with the actual parameters.
I.e.: add `@param` "municipalityId" 

(Comment is applicable on more places in this file)